README.md

```diff
@@ -1 +1,33 @@
-# new_york_times
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> The New York Times Project.
+
+
+
+## Built With
+
+- Html/Css
+- Microverse Linter
+
+
+## Live Demo
+
+[Live Demo Link](https://hamdiheb.github.io/new_york_times/)
+
+
+## Author
+
+👤 **Iheb Hamdi**
+
+- GitHub: [@githubhandle](https://github.com/hamdiheb)
+- Twitter: [@twitterhandle](https://twitter.com/hamdiheb)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hamdi-iheb-b3ba16206/)
```

Comment:
[required] Instead of writing `@githubhandle`, `@twitterhandle`, or `LinkedIn` => replace them with appropriate usernames you use on those social media